_includes/callback-reference.md

```diff
@@ -7,7 +7,7 @@ When a change or update from the back-end system are made on a payment or
 transaction, Swedbank Pay will perform a callback to inform the payee (merchant)
 about this update.
 
-* Setting a `callbackUrl` in the HTTP `POST` API is optional, but highly
+* Setting a `callbackUrl` in the HTTP `POST` API is **mandatory** and highly
```

Comment:
Don't you already have a pull request for this? #320

Reply:
ops this branch should not have this